.ci_support/osx_arm64_.yaml

```diff
@@ -9,20 +9,19 @@ cxx_compiler:
 cxx_compiler_version:
 - '14'
 macos_machine:
-- x86_64-apple-darwin13.4.0
+- arm64-apple-darwin20.0.0
 occt:
 - '7.6'
 pin_run_as_build:
   occt:
     max_pin: x.x
-  python:
-    min_pin: x.x
-    max_pin: x.x
 pybind11_abi:
 - '4'
-python:
-- 3.9.* *_cpython
 target_platform:
-- osx-64
+- osx-arm64
 vtk:
 - 9.2.2
+
+CONDA_BUILD_SYSROOT:
```

Comment:
I have never modified this file directly, do you know if this change persists a re-rendering of the feedstock?